x/mint/keeper/genesis_test.go

```diff
@@ -4,26 +4,183 @@ import (
 	"testing"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/stretchr/testify/require"
-	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
-
-	simapp "github.com/osmosis-labs/osmosis/v7/app"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/stretchr/testify/suite"
 
+	"github.com/osmosis-labs/osmosis/v7/x/mint/keeper"
 	"github.com/osmosis-labs/osmosis/v7/x/mint/types"
 )
 
-func TestMintInitGenesis(t *testing.T) {
-	app := simapp.Setup(false)
-	ctx := app.BaseApp.NewContext(false, tmproto.Header{})
+var customGenesis = types.NewGenesisState(
+	types.NewMinter(sdk.ZeroDec()),
+	types.NewParams(
+		"uosmo",                  // denom
+		sdk.NewDec(200),          // epoch provisions
+		"year",                   // epoch identifier
```

Comment:
why is the epoch identifier set to "year" here?

Reply:
Just to test something other than the default of "day". For every field, I tried to choose something other than default.

My reasoning for this is to make sure that correctness does not depend only on a single possible value of the input.

say I did something like:
```
if epochIdentifier == "day" {
    // correct logic
}
```

Then, if we were to add a different epoch identifier in the future, it wouldn't make sense why it wouldn't work. Testing with at least one different input helps to cover such cases quickly